src/lib/model/OutboundTransfersModel.js

```diff
@@ -789,9 +791,9 @@ class OutboundTransfersModel {
             this._logger.log(`Error running transfer model: ${util.inspect(err)}`);
 
             // as this function is recursive, we dont want to error the state machine multiple times
-            if(this.data.currentState !== 'errored') {
+            if (this.data.currentState !== 'errored') {
```

Comment:
please dont change whitespace unless there is a very good reason to do so.

Reply:
Deleted